cleanrl_utils/huggingface.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import os
+import sys
+from pathlib import Path
+from pprint import pformat
+from typing import List
+
+import numpy as np
+
+HUGGINGFACE_VIDEO_PREVIEW_FILE_NAME = "replay.mp4"
+HUGGINGFACE_README_FILE_NAME = "README.md"
+
+
+def push_to_hub(
+    args: argparse.Namespace,
+    episodic_returns: List,
+    repo_id: str,
+    algo_name: str,
+    folder_path: str,
+    video_folder_path: str = "",
+):
+    # Step 1: lazy import and create / read a huggingface repo
+    from huggingface_hub import CommitOperationAdd, CommitOperationDelete, HfApi
+    from huggingface_hub.repocard import metadata_eval_result, metadata_save
+
+    api = HfApi()
+    repo_url = api.create_repo(
+        repo_id=repo_id,
+        exist_ok=True,
+    )
+    # parse the default entity
+    entity, repo = repo_url.split("/")[-2:]
+    repo_id = f"{entity}/{repo}"
+
+    # Step 2: clean up data
+    # delete previous tfevents and mp4 files
+    operations = []
+    for file in api.list_repo_files(repo_id=repo_id):
+        if ".tfevents" in file or ".mp4" in file:
+            operations += [CommitOperationDelete(path_in_repo=file)]
```

Comment:
(nit) I would use str.endswith("...") to check file extension

```suggestion
    operations = [
        CommitOperationDelete(path_in_repo=file)
        for file in api.list_repo_files(repo_id=repo_id)
        if file.endswith(".tfevents") or file.endswith(".mp4")
    ]
```